SalesforceCommerceCloud/pwa-kit
# [MRT Ref App] Add `MRT_ALLOW_COOKIES` to exposed env vars

Add `MRT_ALLOW_COOKIES` to exposed env vars

# Description

Add `MRT_ALLOW_COOKIES` to exposed env vars so that we can check which environment has it enabled during the CI tests. 

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- update MRT reference app to expose `MRT_ALLOW_COOKIES` env var

# How to Test-Drive This PR

- Locally set MRT_ALLOW_COOKIES to true in env or configure localAllowCookies to true and then see 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
